### PR TITLE
provide additional information in PlaybackOrigin

### PR DESCRIPTION
**Pull request by kkuepper**
Right now we can't exactly know from which playlist a song was played. That makes it difficult to correctly assess the success of our playlists.
Therefore I now include the id and title of podcasts and playlists.

### CODE REVIEW

**Comment by kkuepper** [comment]
@akrol95 What do think of this? Do you have a better idea?

File: BMM.Core/Implementations/DeepLinking/DeepLinkHandler.cs [hunk 1]
```diff
@@ -179,7 +179,9 @@ private bool Open(Uri uri, bool shouldLogEvent)
                 return false;
 
             if (shouldLogEvent)
-                _analytics.LogEvent("deep link opened", new Dictionary<string, object> {{"uri", uri.AbsolutePath}});
+                _analytics.LogEvent("deep link opened", new Dictionary<string, object> { { "uri", uri.AbsolutePath } });
```
Comment (akrol95) [inline BMM.Core/Implementations/DeepLinking/DeepLinkHandler.cs:182]: I would change the name of method parameter from "shouldLogEvent" to  isOpenedFromOutsideOfApp. 
Also I wouldn't do if...else here, but simple 

`_analytics.LogEvent(GetDeepLinkOpenedEventName(isOpenedFromOutsideOfApp), new Dictionary<string, object> {{"uri", uri.AbsolutePath}});`

**Review by akrol95 — APPROVED** [review]

File: BMM.Core/Implementations/DeepLinking/DeepLinkHandler.cs (hunk 1 above)
Comment (kkuepper) [reply BMM.Core/Implementations/DeepLinking/DeepLinkHandler.cs]: refactored

**Comment by kkuepper** [comment]
closes #89

**Comment by kkuepper** [comment]
@akrol95 can you have another look?
On Android the LibraryArchiveFragment does not have a header. Do you want to adjust that?

**Pull request merged**